src/index.html

```diff
@@ -10,20 +10,18 @@
 		<!--- TODO : <meta name="description" content="" />-->
```

Comment:
Don't forget to uncomment and fill this one - it's really important from SEO perspective.